.github/workflows/ci.yaml

```diff
@@ -5,6 +5,12 @@ on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches: [ '**' ]
+    paths:
+      - '**.lock'
+      - '**.toml'
+      - 'node/**'
+      - 'pallets/**'
+      - 'runtime/**'
 
```

Comment:
I think `primitives` folder should be added as well

Reply:
Thanks, updated.